### PR TITLE
Confirm whether current_meta.is_deletion is true or false in logs

### PR DESCRIPTION
**Pull request by gherceg**
## Technical Summary

Followup to https://github.com/dimagi/commcare-hq/pull/32440 as it would be more helpful to log if the delete method is actually called.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/ex-submodules/pillowtop/processors/elastic.py [hunk 1]
```diff
@@ -72,15 +72,27 @@ def process_change(self, change):
             return
 
         if change.deleted and change.id:
-            logger.info(f'Attempting to delete document for change {change.id}')
             doc = change.get_document()
             if doc and doc.get('doc_type'):
+                logger.info(
```
Comment (calellowitz) [inline corehq/ex-submodules/pillowtop/processors/elastic.py:77]: Aren't these the same? The change `id` is the id of the document, and is used by `get_document`.

Comment (gherceg) [reply corehq/ex-submodules/pillowtop/processors/elastic.py:77]: You're absolutely correct 🤦🏻

**Comment by gherceg** [comment]
Revised, and still think this is worth adding to gain insight into this code https://github.com/dimagi/commcare-hq/pull/32680/files#diff-308591ed80f437c63d421731473742a6d8fec03d88c9410b46d1d39704dcc66dR84-R87

**Review by dannyroberts — APPROVED** [review]
This looks good from me. @calellowitz did you have any other blocking feedback?

**Review by calellowitz — APPROVED** [review]
Good from my end too

**Pull request merged**